src/dbnode/persist/fs/types.go

```diff
@@ -309,6 +309,9 @@ type DataFileSetSeekerManager interface {
 	// time, and volume.
 	Return(shard uint32, start xtime.UnixNano, seeker ConcurrentDataFileSetSeeker) error
 
+	// ReturnShard returns (closes) all seekers for a given shard.
+	ReturnShard(shard uint32) error
```

Comment:
Hm, not sure about naming of this. While the `Return(shard, start, seeker)` belongs to `Borrow()`/`Return()` pair, this one simply closes seekers.